Mage/src/main/java/mage/abilities/AbilityImpl.java

```diff
@@ -558,7 +558,11 @@ protected VariableManaCost handleManaXCosts(Game game, boolean noMana, Player co
                     } else {
                         String manaSymbol = null;
                         if (variableManaCost.getFilter().isBlack()) {
-                            manaSymbol = "B";
+                            if (variableManaCost.getFilter().isRed()) {
+                                manaSymbol = "B/R";
```

Comment:
The issue is two fold here:
- this code should actually not be here in the first place. So I can see why SickBoyWi extended the if-statements here.  I would rather see this as a method on VariableManaCost. Perhaps that can be a follow up task after this?
- adding hybridmana is obviously a new requirement here, can we make a mapping to the Enum ManaSymbol perhaps?